types/modules/input.pp

```diff
@@ -6,6 +6,7 @@
   'imkafka',
   'imklog',
   'imkmsg',
+  'immark',
```

Comment:
Where does `immark` come from? I don't see it in the list in the Rsyslog documentation?

Reply:
https://github.com/rsyslog/rsyslog-doc/pull/830